models/lorenz_96/work/path_names_preprocess

```diff
@@ -1,4 +1,5 @@
 assimilation_code/modules/utilities/null_mpi_utilities_mod.f90
```

Comment:
I'll revert it.  Does obs_seq.out need to be under version control?

Reply:
great question.  it's always been in the repo since i can remember, but for L96 our instructions (and workshop scripts) always run PMO first, which generates it.  maybe i'm forgetting some place it's needed?  but removing it would save a lot of commit errors.  i bet if you look at this poor file's history, it's been mis-committed and reverted more times than any other file in the system.

Reply:
I've reverted obs_seq.out.  Adding an issue to look at whether this needs to be under version control (it is ~6.6MB)